packages/react/src/component-tests/IcAlert/IcAlert.cy.tsx

```diff
@@ -0,0 +1,344 @@
+      .should("be.visible")
+      .contains("Warning");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .contains("This alert is for displaying warnings.")
+      .should("be.visible");
+  });
+
+  it("Success Alert with Message and Icon", () => {
+    mount(
+      <IcAlert
+        variant="success"
+        heading="Success"
+        message="This alert is for displaying success messages."
+      />
+    );
+    cy.checkHydrated("ic-alert");
+    cy.findShadowEl("ic-alert", "svg").should("be.visible");
+    cy.get('[heading="Success"]').should("be.visible");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .find("p")
+      .should("be.visible")
+      .contains("Success");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .contains("This alert is for displaying success messages.")
+      .should("be.visible");
+  });
+
+  it("should close on dismiss icon", () => {
+    mount(
+      <IcAlert
+        heading="Dismissible"
+        message="This alert is dismissible."
+        dismissible
+      />
+    );
+    cy.checkHydrated("ic-alert");
+    cy.get("ic-alert").should("exist");
+    cy.findShadowEl("ic-alert", "ic-button").should("be.visible").click();
+  });
+
+  it("Responsive title with Message and Icon", () => {
+    mount(
+      <IcAlert
+        heading="This title is very long so should force the alert to add 'title-above'
+    "
+        message="This alert does not have 'title-above' added as a prop but it's forced to add it due to the length of the title."
+      />
+    );
+    cy.checkHydrated("ic-alert");
+    cy.findShadowEl("ic-alert", "svg").should("be.visible");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .find("p")
+      .should("be.visible")
+      .contains(
+        "This title is very long so should force the alert to add 'title-above"
+      )
+      .should("be.visible");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .contains(
+        "This alert does not have 'title-above' added as a prop but it's forced to add it due to the length of the title."
+      )
+      .should("be.visible");
+  });
+
+  it("Title displayed with Message and Icon", () => {
+    mount(
+      <IcAlert
+        heading="Title displayed above the message"
+        message="This alert has the title at the top of the alert and the message below it."
+        title-above="true"
+      />
+    );
+    cy.checkHydrated("ic-alert");
+    cy.findShadowEl("ic-alert", "svg").should("be.visible");
+    cy.get('[heading="Title displayed above the message"]').should(
+      "be.visible"
+    );
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .find("p")
+      .should("be.visible")
+      .contains("Title displayed above the message");
+    cy.findShadowEl("ic-alert", "ic-typography")
+      .contains(
+        "This alert has the title at the top of the alert and the message below it."
+      )
+      .should("be.visible");
+  });
+
+  it.s("Custom with Message and Icon", () => {
```

Comment:
```suggestion
  it("Custom with Message and Icon", () => {
```
this is causing tests to fail to run for me